zulipterminal/core.py

```diff
@@ -300,6 +301,10 @@ def show_stream_info(self, stream_id: int) -> None:
         show_stream_view = StreamInfoView(self, stream_id)
         self.show_pop_up(show_stream_view, "area:stream")
 
+    def show_topic_info(self, stream_id: int, topic_name: str) -> None:
+        show_topic_view = TopicInfoView(self, stream_id, topic_name)
+        self.show_pop_up(show_topic_view, "area:stream")
```

Comment:
Similarly to the help, we may want to update the label here, or add a new one if want a new area - though we risk running out of styles :)